docs/our_work/ratings-and-reviews.md

```diff
@@ -24,7 +24,7 @@ The flow follows four stages:
 
 4. The Flask app now has an answer for each of the rules which are to be applied, and can send a final response to the NHS.UK website.
 
-If the review doesn't violate any of these, the app will tell NHS.UK that the review is all good, and it will be published. If there's anything amiss, the website will take a response which depends upon which rule wasn't met. This ranges from asking the reviewer to remove something (such as a name) from a review, to directing them to resources specific to the content of their review. For more information on these pathways, see !!!!INSERT LINK!!!.
+I think this might be clearer "If there's anything amiss, the website will produce a response informing the user (the person leaving the review) which rule has been broken. This ranges from asking the reviewer to remove something (such as a name) from a review, to directing them to resources specific to the content of their review.
```

Comment:
Sorry just spotted this. The start of this sentence needs removing and the double quotes